src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -859,6 +860,94 @@ File 1
         }
 
         [Fact]
+        public void LocalFunctionWithLambdaCoverage()
+        {
+            string source = @"
+using System;
+
+public class Program
+{
+    public static void Main(string[] args)                                  // Method 1
+    {
+        TestMain();
+        Microsoft.CodeAnalysis.Runtime.Instrumentation.FlushPayload();
+    }
+    
+    static void TestMain()                                                  // Method 2
+    {
+        new D().M1();
+    } 
+}
+
+public class D
+{
+    public void M1()                                                        // Method 4
+    {
+        L1();
+        void L1()
+        {
+            var f = new Func<int>(
+                () => 1
+            );
+
+            var f1 = new Func<int>(
+                () => 1
+            );
+
+            f();
+        }
+    }
+
+    // Method 5 is the synthesized instance constructor for D.
+}
+";
+
+            string expectedOutput = @"Flushing
+Method 1
+File 1
+True
+True
+True
+Method 2
+File 1
+True
+True
+Method 4
+File 1
+True
+True
+True
+True
+False
```

Comment:
Sorry to bring this up again, but all those instrumentation tests still suffer a significant readability problem. I'm trying to figure out which statement this `false` comes from and it is confusing and error-prone.
Any ideas how the recorded spans could be displayed back?

Would it be possible to output the original source code with inserted trivia representing the instrumentation result?
```C#
/* Method 1, File 1 */ public static void Main(string[] args)
    {
        /* True */ TestMain();
        /* True */ Microsoft.CodeAnalysis.Runtime.Instrumentation.FlushPayload();
        // Note I don't remember where the third recorded flag belong...
    }
````